vite.config.ts

```diff
@@ -20,7 +20,8 @@ export default defineConfig({
         provider: "c8",
         reporter: ['text', 'lcov']
     },
-    exclude: [...configDefaults.exclude, "service/**"]
+    exclude: [...configDefaults.exclude, "service/**"],
+    environment: 'jsdom'
```

Comment:
I'd prefer to keep the default environment as `node` and use the `jsdom` environment only for individual UI-oriented files, as with the [ChainsSelector.test.tsx](https://github.com/flatironinstitute/mcmc-monitor/blob/0f79b0b0b0f9a19e49ad5f85333f4c4f27062def/test/components/ChainsSelector.test.tsx#L1) example.

Also, depending on the circumstance, these issues can occur during the import chain for logic that we actually want to mock out anyway; see for instance the mocks of `config` and `fetch` in the [postApiRequest.test.ts](https://github.com/flatironinstitute/mcmc-monitor/blob/0f79b0b0b0f9a19e49ad5f85333f4c4f27062def/test/networking/postApiRequest.test.ts) tests.

If you want to call out specific things that were failing, I can take a look after my meeting and make some suggestions?